man/systemd.network.xml

```diff
@@ -2350,6 +2350,32 @@ IPv6Token=prefixstable:2002:da8:1::</programlisting></para>
     </variablelist>
   </refsect1>
 
+  <refsect1>
+      <title>[DHCPServerStaticLease] Section Options</title>
+      <para>The <literal>[DHCPServerStaticLease]</literal> section configures a static DHCP lease
+      to assign a pre-set IPv4 address to a specific device based on its MAC address.
+      This section can be specified multiple times.
+      </para>
+
+      <variablelist class='network-directives'>
+        <varlistentry>
+          <term><varname>MACAddress=</varname></term>
+
+          <listitem><para>The hardware address of a device which should be assigned IPv4 address
+          specified in <varname>Address=</varname>. This key is mandatory.</para>
+          </listitem>
+        </varlistentry>
+
+        <varlistentry>
+          <term><varname>Address=</varname></term>
+
+          <listitem><para>IPv4 address that should be assigned to a device with a hardware address
```

Comment:
There is no mention here of whether the `Address` must be in the pool, or in the subnet.  Looking in the code, i also couldn't figure out the answer (but i only did a brief skim).  I think an address that is outside of the subnet is very likely to cause trouble (because things like netmask and gateway would probably be wrong), and therefore should probably be explicitly rejected.  But an address inside the subnet but *outside* of the pool is legitimate and probably commonplace (e.g. the pool starts at `x.y.z.100` and static addresses are < `x.y.z.100`).

At the very least, this should describe the constraints for the `Address`.